.github/workflows/test.yml

```diff
@@ -0,0 +1,20 @@
+name: Parser compilation and tests
+
+on: [push, pull_request]
+
+jobs:
+  check_compilation_with_cl:
+    name: Parser compilation using MSVC cl.exe
+    runs-on: 'windows-latest'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ilammy/msvc-dev-cmd
+
+      - name: Install and prepare Neovim
+        run: |
+          bash ./scripts/ci-install-windows-latest.sh
+          echo "C:\\tools\\neovim\\Neovim\\bin\\" >> $GITHUB_PATH
+
+      - name: Compile parsers
+        run: nvim --headless -c "TSInstall all" -c "sleep 120" -c "q"
```

Comment:
Does `:TSSync all` work? Then `sleep` would not be necessary.

Reply:
apparently it works :)